website/docs/docs-section/roadmap/v3/major-changes.md

```diff
@@ -4,28 +4,30 @@ title: Major Change from V2
 sidebar_label: Major Changes
 ---
 
-Here are some of the major changes from OpenAttestation v2 Data Model
+This article describes some major changes from the OpenAttestation V2 Data Model.
 
 ## `@context`
 
-The `@context` property is now introduced as per [W3C VC Care Data Model](https://www.w3.org/TR/vc-data-model/#contexts) to allow the issuer to map such short-form aliases to the URIs required by specific verifiable credentials and verifiable presentations.
+To help the issuer map short-form aliases to the URIs required by specific verifiable credentials and verifiable presentations, the `@context` property is introduced as per [W3C VC Care Data Model](https://www.w3.org/TR/vc-data-model/#contexts).
```

Comment:
From a changelog perspective it would be more useful to first see what has been changed followed by the rationale.

Reply:
OK, updated. Also changed the headings to be easier to read.